lib/propshaft/server.rb

```diff
@@ -34,8 +34,9 @@ def inspect
   private
     def extract_path_and_digest(env)
       full_path = Rack::Utils.unescape(env["PATH_INFO"].to_s.sub(/^\//, ""))
-      digest    = full_path[/-([0-9a-zA-Z]{7,128})\.(?!digested)[^.]+\z/, 1]
-      path      = digest ? full_path.sub("-#{digest}", "") : full_path
+      digest = full_path[/-([0-9a-zA-Z]{7,128})\.(?!digested)[^.]+\z/, 1]
+      digest ||= full_path[Propshaft::Asset::PREDIGESTED_REGEX, 1]
+      path = digest ? full_path.sub("-#{digest}", "") : full_path
```

Comment:
Same as I mentioned in `asset.rb`: we can probably drop the `.digested` suffix here.